packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -106,8 +102,8 @@ const ClinicalResultBloodPressure = (props) => {
     let datetimeDisplayElement = null;
     const decoratedResultDisplay = [];
 
-    const hasSystolic = resultData.systolic;
-    const noDataSystolic = (hasSystolic && resultData.systolic.resultNoData === true);
+    const hasSystolic = systolic;
```

Comment:
We shouldn't be reassinging an object to a boolean named constant. This might be more appopraite to check if a value was provided:
```
const hasSystolic = !!systolic;
```